frontend/src/common/utils/checkForRevisionChange.ts

```diff
@@ -35,6 +35,22 @@ function checkListForChanges(
   return !isEmpty(xorWith(revisedList, publishedList, isEqual));
 }
 
+/**
```

Comment:
Local dataset order diff check: required to determine if publish button should be enabled.